docs/user-guide/garbage-collector.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+* TOC
+{:toc}
+
+## WARNING: Garbage Collector is an alpha feature, use it at your own risk!
+
+### What does the garbage collector do

```

Comment:
This section has a lot of implementation detail.  Start with what is meaningful to the user, and put implementation detail in a separate section.  What is the Purpose? - e.g. to delete resources created by delete controllers.